egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -116,55 +96,158 @@ impl PlotDemo {
+struct MarkerDemo {
+    fill_markers: bool,
+    marker_radius: f32,
+    custom_marker_color: bool,
+    marker_color: Color32,
+}
+
+impl Default for MarkerDemo {
+    fn default() -> Self {
+        Self {
+            fill_markers: true,
+            marker_radius: 5.0,
+            custom_marker_color: false,
+            marker_color: Color32::GRAY,
+        }
+    }
+}
+
+impl MarkerDemo {
+    fn markers(&self) -> Vec<Points> {
+        MarkerShape::all()
+            .into_iter()
+            .enumerate()
+            .map(|(i, marker)| {
+                let y_offset = i as f32 * 0.5 + 1.0;
+                let mut points = Points::new(ValueSeries::from_values(vec![
+                    Value::new(1.0, 0.0 + y_offset),
+                    Value::new(2.0, 0.5 + y_offset),
+                    Value::new(3.0, 0.0 + y_offset),
+                    Value::new(4.0, 0.5 + y_offset),
+                    Value::new(5.0, 0.0 + y_offset),
+                    Value::new(6.0, 0.5 + y_offset),
+                ]))
+                .name("Marker Lines")
+                .filled(self.fill_markers)
+                .radius(self.marker_radius)
+                .shape(marker);
+
+                if self.custom_marker_color {
+                    points = points.color(self.marker_color);
+                }
+
+                points
+            })
+            .collect()
+    }
+}
+
+impl Widget for &mut MarkerDemo {
+    fn ui(self, ui: &mut Ui) -> Response {
+        ui.horizontal(|ui| {
+            ui.checkbox(&mut self.fill_markers, "fill markers");
+            ui.add(
+                egui::DragValue::new(&mut self.marker_radius)
+                    .speed(0.1)
+                    .clamp_range(0.0..=f32::INFINITY)
+                    .prefix("marker radius: "),
+            );
+            ui.checkbox(&mut self.custom_marker_color, "custom marker color");
+            if self.custom_marker_color {
+                ui.color_edit_button_srgba(&mut self.marker_color);
+            }
+        });
+
+        let mut markers_plot = Plot::new("Markers Demo").height(300.0).data_aspect(1.0);
+        for marker in self.markers() {
+            markers_plot = markers_plot.points(marker);
+        }
+        ui.add(markers_plot)
+    }
+}
+
+#[derive(PartialEq, Default)]
+pub struct PlotDemo {
+    line_demo: LineDemo,
+    marker_demo: MarkerDemo,
+}
+
+impl super::Demo for PlotDemo {
+    fn name(&self) -> &'static str {
+        "🗠 Plot"
+    }
+
+    fn show(&mut self, ctx: &CtxRef, open: &mut bool) {
+        use super::View;
+        Window::new(self.name())
+            .open(open)
+            .default_size(vec2(400.0, 400.0))
+            .scroll(false)
+            .show(ctx, |ui| self.ui(ui));
+    }
+}
+
+impl super::View for PlotDemo {
+    fn ui(&mut self, ui: &mut Ui) {
+        ui.vertical_centered(|ui| {
+            egui::reset_button(ui, self);
+            ui.add(crate::__egui_github_link_file!());
+        });
```

Comment:
Maybe the "Pan by dragging…" instructions could be moved here, since it is common for both the lines and markers demo